chcemvediet/apps/obligees/tests/test_views.py

```diff
@@ -60,6 +60,7 @@ def test_paginator_with_invalid_page_number_shows_first_page(self):
         self.assertEqual(list(response.context[u'obligee_page']), oblgs[:25])
 
     def test_paginator_with_no_obligees(self):
+        Obligee.objects.all().delete()
```

Comment:
@viliambalaz Tu ti ostala nejaka zmena v `test_views.py` na prvy pohlad nesuvisiaca s `test_models.py` testami, ktore riesime v tomto PR. Ci je to nieco, co tu treba v tomto PR?

Reply:
Spolu suvisi s https://github.com/martinmacko47/chcemvediet/pull/305#pullrequestreview-506115992. Asi to nebolo stastne riesenie, kedze tieto data budeme potrebovat mozno v dalsich testoch. Asi by bolo lepsie ich nejako odfiltrovat, nanovo vytvorit po skonceni testu, pouzit druhu databazu alebo vytvorit nejaky mock.

Reply:
@viliambalaz Ok. To vyriesime neskor, ked budeme riesit `obligees/tests/test_views.py`. Zatial tuto zmenu vyhod a pridaj tam `@unittest.skip(u'FIXME')`

Reply:
:ok: